lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -54,7 +54,7 @@ class Cookbook < Base
           long:         "--delivery",
           description:  "Generate cookbook with delivery integration",
```

Comment:
Let's update this with a message stating that this behavior is the default and the option has no effect.